### PR TITLE
Fixes #328

### LINKED ISSUE #328: Podman tag is missing?
Hi,

Is there a way to create a tag for an existing image?

```
podman tag my.repo.domain.com/app:1.2.3 app:1.2.3
```

I had a look at the existing modules and I didn't see a dedicated one for tagging purposes...

### PR DESCRIPTION
**Pull request by ocafebabe**

### CODE REVIEW

File: plugins/modules/podman_tag.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# Copyright (c) 2021, Sagi Shnaidman <sshnaidm@redhat.com>
```
Comment (sshnaidm) [inline plugins/modules/podman_tag.py:4]: Thanks, but you can put your own copyright :)

File: tests/integration/targets/podman_tag/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  always:
+    - name: Cleanup image
+      containers.podman.podman_image:
+        name: "{{ item }}"
+        state: absent
+      loop:
+        - docker.io/library/openjdk:8-jdk-alpine
+        - localhost/openjdk8
+        - localhost/jdk8
```
Comment (sshnaidm) [inline tests/integration/targets/podman_tag/tasks/main.yml:36]: nit: need an end line

**Review by sshnaidm — APPROVED** [review]

File: tests/integration/targets/podman_tag/tasks/main.yml (hunk 2 above)
Comment (ocafebabe) [reply tests/integration/targets/podman_tag/tasks/main.yml:36]: Do you want another PR for that?

Comment (sshnaidm) [reply tests/integration/targets/podman_tag/tasks/main.yml:36]: I fixed it

**Pull request merged**